CHANGES.rst

```diff
@@ -262,6 +262,11 @@ astropy.io.fits
 astropy.io.misc
 ^^^^^^^^^^^^^^^
 
+- Fix Tables with columns of strings can't be written to Hdf5 file with Python3
```

Comment:
Can you reword this as ``Fix a bug that caused tables with columns of strings to not be written to HDF5 with Python 3.x``